oscarmorrison/react-webpack-starter
# move eslint, babel to separate dot files

Fixes #12

**Changes**  
- move eslintrc, babelrc to separate dot files
 
**To Verify**    
- checkout branch `git checkout <branch-name>`
- build `yarn build`
- verify
